smithy-typescript-ssdk-libs/server-node/package.json

```diff
@@ -0,0 +1,67 @@
+{
+  "name": "@aws-smithy/server-node",
+  "version": "1.0.0-alpha.8",
+  "description": "Base components for Smithy services running on NodeJS",
+  "main": "./dist/cjs/index.js",
+  "module": "./dist/es/index.js",
+  "types": "./dist/types/index.d.ts",
+  "scripts": {
+    "prepublishOnly": "yarn build",
+    "pretest": "yarn build",
+    "build": "concurrently 'yarn:build:cjs' 'yarn:build:es' 'yarn:build:types'",
+    "build:cjs": "tsc -p tsconfig.cjs.json",
+    "build:es": "tsc -p tsconfig.es.json",
+    "build:types": "tsc -p tsconfig.types.json",
+    "postbuild": "rimraf dist/types/ts3.4 && downlevel-dts dist/types dist/types/ts3.4",
+    "test": "jest --passWithNoTests",
+    "clean": "rimraf dist",
+    "lint": "yarn run eslint -c ../.eslintrc.js \"src/**/*.ts\"",
+    "format": "prettier --config ../prettier.config.js --ignore-path ../.prettierignore --write \"**/*.{ts,md,json}\""
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/awslabs/smithy-typescript.git",
+    "directory": "smithy-typescript-ssdk-libs/server-node"
+  },
+  "author": "AWS Smithy Team",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@aws-sdk/node-http-handler": "^3.272.0",
+    "@aws-sdk/protocol-http": "^3.267.0",
+    "@aws-sdk/smithy-client": "^3.279.0",
+    "@aws-sdk/types": "^3.267.0",
+    "@aws-sdk/util-base64-node": "^3.209.0",
+    "@aws-sdk/util-body-length-node": "^3.208.0",
+    "@aws-sdk/util-utf8-node": "^3.259.0",
+    "@types/aws-lambda": "^8.10.72",
```

Comment:
seems like you may not need most of these expect `@aws-sdk/types` and `@aws-sdk/protocol-http`

Reply:
Yes, you are correct here. I was doing some experiments adapting the Lambda/API Gateway handler code and it seems I forgot to remove them. Thanks!